core/drivers/stm32_iwdg.c

```diff
@@ -358,7 +395,8 @@ static TEE_Result iwdg_wdt_get_timeleft(struct wdt_chip *chip, bool *is_started,
 	if (res)
 		panic();
 
-	time.seconds = iwdg->timeout;
+	time.seconds = (iwdg->timeout - IWDG_ETIMEOUT_SEC) * iwdg->saved_nb_int
```

Comment:
I think `iwdg->early_timeout` should be used in place of `IWDG_ETIMEOUT_SEC` here.